root/event/edit_form.tt

```diff
@@ -20,7 +20,8 @@
         [% END %]
         <p>
             [%- l('Add "@ " at line start to indicate artists, "* " for a work/song, "# " for additional info (such as "Encore").
-                   [mbid|name] allows linking to artists and works.') -%]
+                   [mbid|name] allows linking to artists and works.
+                   Escape the symbol "[" using the HTML entity "&lrqb;", "]" using "&lsqb;", "&" using "&amp;".') -%]
```

Comment:
Written like this it sounds a bit like you need to escape [] in [mbid|name] to me, but maybe it's not confusing for people? We could prepend "To show some symbols as part of the setlist text you will need to escape them:"